src/utils/components/CapacityInput/useUnitOptions.ts

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react';
+
+import { toQuantity } from '@kubevirt-utils/utils/units';
+
+import { getUnitOptions } from './utils';
+
+const useUnitOptions = (size: string) => {
+  const [initialSize] = useState(size);
+  const { unit: initialUnit } = toQuantity(initialSize);
+
+  return getUnitOptions(initialUnit);
```

Comment:
why we have `useState` if we consider only the initialSize and don't set anymore the state?